plugins/search/src/components/SearchPage/SearchPage.test.tsx

```diff
@@ -75,13 +84,11 @@ describe('SearchPage', () => {
     // When we render the page...
     await renderInTestApp(<SearchPage />);
 
-    // Then search context should be initialized with these values...
-    const calls = (SearchContextProvider as jest.Mock).mock.calls[0];
-    const actualInitialState = calls[0].initialState;
-    expect(actualInitialState.term).toEqual(expectedTerm);
-    expect(actualInitialState.types).toEqual(expectedTypes);
-    expect(actualInitialState.pageCursor).toEqual(expectedPageCursor);
-    expect(actualInitialState.filters).toStrictEqual(expectedFilters);
+    // Then search context should be set with these values...
+    expect(setTermMock.mock.calls[0][0]).toEqual(expectedTerm);
```

Comment:
sounds like these could be on the form `expect(mock).toHaveBeenCalledWith(value)` instead